content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -24,35 +27,35 @@ Developed by the Centre for Infectious Disease Genomics and One Health (CIDGOH)
 
 
 [![](../_static/images/howto_guides/portal_guide/ORCiD.png)](../_static/images/howto_guides/portal_guide/ORCiD.png)
 
-NMDC Products require ORCiD authentication for full access and functionality.
+NMDC resources require ORCiD authentication for full access and functionality.
 
 ## Submission Portal Functionality
 
 [https://data.microbiomedata.org/submission/home](https://data.microbiomedata.org/submission/home) 
 
-The Submission Portal requires ORCiD authentication to access. If you have already signed in via ORCID, you will not see this login screen in the NMDC Submission Portal.
+The Submission Portal requires ORCiD authentication to access. If you have already signed in via ORCID, you will not see this login screen within the Submission Portal.
 
 [![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)
 
-Once signed in with an ORCiD, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to NMDC. Details about each section are outlined below. Submitters can return to existing submissions saved under the ORCiD account to resume their work at any point.   
+Once signed in with an ORCiD, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to the NMDC. Details about each section are outlined below. Submitters can return to existing submissions saved under the ORCiD account to resume their work at any point.   
 
 ### Submission Context
 
 [![](../_static/images/howto_guides/portal_guide/submission_context.png)](../_static/images/howto_guides/portal_guide/submision_context.png)
 
-On the Submission Context screen, submitters are asked whether data has already been generated for their study.  If a user selects ‘Yes’ a DOI associated with the data can be entered. For data generated at either JGI or EMSL (DOE user facilities), the specific Award DOI (e.g., 10.46936/10.25585/60001289) should be entered, along with selecting the checkbox for this option. If the data was not generated at a DOE User Facility, a valid data DOI can be entered. A data DOI is not the same as a publication DOI, and would be issued through a separate resource as a unique persistent identifier (e.g., 10.48443/e4zf-b917). 
+On the Submission Context screen, submitters are asked whether data has already been generated for their study.  If a user selects ‘Yes’ a DOI associated with the data can be entered. For data generated at either the JGI or EMSL, the specific Award DOI (e.g., 10.46936/10.25585/60001289) should be entered, along with selecting the checkbox for this option. If the data was not generated at a DOE User Facility, a valid data DOI can be entered. A data DOI is not the same as a publication DOI, and would be issued through a separate resource as a unique persistent identifier (e.g., 10.48443/e4zf-b917). 
```

Comment:
Inconsistent us of "you" or "submitter" or "users"

Pick one and make it consistent. I suggest "submitter"

Reply:
Will submit an issue and make this change in a later PR

Reply:
+1 for "submitters" or "users". "you" feels a bit weird